runtime/gc_base/GCExtensions.cpp

```diff
@@ -338,3 +341,20 @@ MM_GCExtensions::needScanStacksForContinuationObject(J9VMThread *vmThread, j9obj
 #endif /* JAVA_SPEC_VERSION >= 19 */
 	return needScan;
 }
+
+bool
+MM_GCExtensions::isConcurrentScavengerInActive(MM_EnvironmentBase* env)
```

Comment:
to be consistent with naming of MM_Scavenger::isCurrentPhaseConcurrent() let's call this method: isCurrentScavengerPhaseConcurrent()